### DISCUSSION TITLE
(Unintentionally?) Invalidated default key bindings

### DISCUSSION THREAD

**Discussion by Igonato** [jellydn/zed-101-setup#60] [Ideas]
### Describe the bug

Some default keybindings  no longer work after using keybindings  from the readme:

 - `ctrl-k cltr-s` - open keymap
 - `ctrl-k ctrl-t` - open theme switcher
 - etc...

But they still show up in the command palette causing confusion.

I suggest explicitly disabling them:

```json
"bindings": {
  "ctrl-k ctrl-s": null
}
```

and/or defining an alternative.

**Comment by jellydn** [comment]
I don't have an issue with the default keymap in my settings. I'm on Mac by the way.
